Source/WebInspectorUI/UserInterface/Views/FolderTreeElement.js

```diff
@@ -25,11 +25,36 @@
 
 WI.FolderTreeElement = class FolderTreeElement extends WI.GeneralTreeElement
 {
-    constructor(title, representedObject)
+    constructor(title, representedObject, id = null)
```

Comment:
Style: we prefer using `options = {}` options "bags" instead of parameters with default values as it's _much_ easier to extend in the future
```
constructor(title, representedObject, options = {})
```